canonical-websites/snapcraft.io
# Updates to releases confirmation

Fixes #1398 
Fixes #1424 

- Removes (?) icon from release confirmation. Shows tooltips when hover over releases/closes count.
- Fixes the issue when promoting into channel pending close (#1424)
- Moves closed channels state into Redux

### QA
- ./run or demo
- go to Releases page of any snap
- promote some channel, close some channel
- There should be no (?) icon in confirmation box, hovering over 'X releases' or 'X closes' should show tooltips with details
- Try to reproduce issue #1424 (promote something into channel that is pending close) - it shouldn't be possible anymore

<img width="1029" alt="screen shot 2018-12-11 at 13 20 38" src="https://user-images.githubusercontent.com/83575/49800353-950d9200-fd47-11e8-8595-ce5efed1af8a.png">
